ail_vru_uc1_scenario/README.md

```diff
@@ -18,9 +18,13 @@ This CARMA Configuration Image creates a **AIL** (Anything-In-the-Loop) scenario
 | SUMO      | 1.15       |
 
 ## Deployment Instructions
-
+### Deployment Steps
+TODO: Complete this section
```

Comment:
We should flesh this out before merging.

Reply:
I do not think so. This is more related to the entire deployment than this data collection.